pycode/memilio-epidata/memilio/epidata/getNPIData.py

```diff
@@ -137,136 +139,158 @@ def read_files(directory, fine_resolution, run_checks):
+                            if counter_codes == 0:
+                                counties = np.sort(
+                                    df_npis_per_code.ags5.unique())
+                                num_counties = len(
+                                    df_npis_per_code.ags5.unique())
+
+                                # extract dates from data
+                                dates = df_npis_per_code.iloc[:int(
+                                    df_npis_per_code.shape[0]/num_counties), 5]
+                                # rename dates so that they match dates from other npi dataframe
+                                dates_new = [
+                                    'd' + date.replace('-', '') for date in dates]
+
+                                df_local = [pd.DataFrame()
+                                            for i in range(num_counties)]
+
+                            #  set df for all counties
+                            for i in range(0, num_counties):
+                                if counter_codes == 0:
+                                    df_local[i] = pd.DataFrame(
+                                        columns=list(df_npis_per_code.columns[0: 5]) +
+                                        ['code'] + dates_new)
+
+                                dummy_to_append = pd.DataFrame(
+                                    columns=['code'] + dates_new,
+                                    data=copy.deepcopy(df_npis_per_code
+                                                       [df_npis_per_code.ags5 == counties[i]].
+                                                       iloc[:, 6:].T.reset_index().values))
+
+                                df_local[i] = pd.concat(
+                                    [df_local[i], dummy_to_append])
+
+                                if df_npis_per_code.iloc[i * len(dates): (i + 1) *
+                                                         len(dates),
+                                                         3].nunique() > 1:
+                                    raise gd.DataError(
+                                        'Dates are not sorted as expected.')
+
+                                # Set first five columns so that they match old format of data frame (from kr_massnahmen_unterkategorien.csv)
+                                if counter_codes == len(codelist)-1:
+                                    df_local[i][df_local[i].columns[0:5]
+                                                ] = df_npis_per_code.iloc[i*len(dates), 0:5].values
+
+                            counter_codes += 1
+                        df_npis_old = pd.concat([df_local[i]
+                                                for i in range(num_counties)])
+                        # 'bundesland' maps to stateID for DIVI, so rename it seperately here
+                        df_npis_old.rename(
+                            {'bundesland': dd.EngEng('state')}, axis=1, inplace=True)
```

Comment:
When executing the script with fine_resolution=1 I get the error `TypeError: 'dict' object is not callable`